ttnn/cpp/ttnn/operations/sliding_window/sliding_window.hpp

```diff
@@ -48,6 +48,7 @@ struct SlidingWindowConfig {
     CoreRangeSet core_range_set = std::set{CoreRange({0, 0}, {0, 0})};   // active cores
 
     bool snap_to_tile = false;
+    bool is_bilinear = false;
 
```

Comment:
Why does this exist?

Reply:
we are treating bilinear as special case where input and output is equal for kernel_size 2x2. SInce the conv output computation it will either be output = (input -1)/ (input + 1) for padding 0 and 1, we have added this variable to force the output to be equal.